packages/components/src/internal/components/entities/EntityInsertGridRequiredFieldAlert.tsx

```diff
@@ -46,7 +46,7 @@ export const EntityInsertGridRequiredFieldAlert: FC<Props> = memo(props => {
 // exported for jest testing
 export function getFieldKeysOfRequiredCols(cols: List<QueryColumn>): string[] {
     return cols
-        .filter(col => col.isEditable() && col.required)
+        .filter(col => col.isEditable() && col.required && col.fieldKeyArray.length === 1)
```

Comment:
at first I was thinking "why not just look to see if col.lookup" but this is the field from the lookup, not the lookup itself. This fieldKeyArray length check makes sense then.